app/test/unit/git/stash-test.ts

```diff
@@ -0,0 +1,32 @@
+import * as FSE from 'fs-extra'
+import * as path from 'path'
+import { Repository } from '../../../src/models/repository'
+import { setupEmptyRepository } from '../../helpers/repositories'
+import { GitProcess } from 'dugite'
+import { MagicStashString, getStashEntries } from '../../../src/lib/git/stash'
+
+describe('git/stash', () => {
+  describe('getStashEntries', () => {
+    it('returns all stash entries created by desktop', async () => {
```

Comment:
I'm planning to add that in the PR I'll be opening that adds the functionality to create a stash entry since I likely will have settled on a message format to use. At the moment, I'm splitting on `:` simply because the test repo I've been working in uses that format.

Reply:
I went ahead and added this functionality here: https://github.com/desktop/desktop/pull/7032/commits/90d568c212494f4503b6db2e4b7b3b1825e9eee2